tutormfe/patches/mfe-dockerfile-pre-npm-build-account

```diff
@@ -0,0 +1,4 @@
+RUN git config --global user.email "tutor@overhang.io" \
+  && git config --global user.name "Tutor"
```

Comment:
nit: do we need to configure git user here?

Reply:
Yes, we need to do so. We are getting below error if we aren't configuring git user:
```bash
1.542 Committer identity unknown
1.542 
1.542 *** Please tell me who you are.
1.542 
1.542 Run
1.542 
1.542   git config --global user.email "you@example.com"
1.542   git config --global user.name "Your Name"
1.542 
1.542 to set your account's default identity.
1.542 Omit --global to set the identity only in this repository.
1.542 
1.542 fatal: unable to auto-detect email address (got 'root@buildkitsandbox.(none)')
------
Dockerfile:113
--------------------
 111 |     ARG ENABLE_NEW_RELIC=false
 112 |     
 113 | >>> RUN curl -fsSL https://github.com/openedx/frontend-app-account/commit/16fb3a1bb4c00d2b60ec3686001814461437e3a5.patch | git am
 114 |     
 115 |     ######## account (dev)
```